mod/caching/set/abstract/tagged_by_cached_count.rb

```diff
@@ -22,8 +24,18 @@ def self.included host_class
     end
 
     define_method :wql_from_content do
-      { type_id: Card::Codename.id(host_class.type_to_count),
-        right_plus: [Card::Codename.id(host_class.tag_pointer), { refer_to: left.id }] }
+      { type_id: Card::Codename.id(host_class.type_to_count), right_plus: right_plus_val }
+    end
+
+    define_method :right_plus_val do
+      [Card::Codename.id(host_class.tag_pointer), { refer_to: left.id }]
     end
   end
 end
+
+# FIXME: hack. otherwise filter wql can overwrite right_plus
+def filter_wql
```

Comment:
I don't love this.  Would love to discuss longer term plans for wql/cql that include solutions for the unique key problem.